### ISSUE TITLE
FusionCharts 3.0.1 requires Angular 4 and rxjs 5

### ISSUE THREAD

**Issue by luiscmas** [fusioncharts/angular-fusioncharts#55]
Hi, last version it still requires Angular 4 and rxjs 5, when today Angular 7 and rxjs 6 are the current versions.

**Comment by ashok1994** [comment]
Hi @luiscmas  , though the dependencies are Angular 4 and rxjs 5 , they are completely  compatible with the latest version.
If you are facing any problem , please lets us know

**Comment by andrusaru** [comment]
Hi, it is still incompatible with Angular 8+

**Comment by AyanBhadury** [comment]
Yes please download the latest version  : http://www.npmjs.com/package/angular-fusioncharts
@andrusaru
